lib/ts/recipe/session/sessionFunctions.ts

```diff
@@ -34,6 +34,12 @@ export async function createNewSession(
     accessTokenPayload = accessTokenPayload === null || accessTokenPayload === undefined ? {} : accessTokenPayload;
     sessionData = sessionData === null || sessionData === undefined ? {} : sessionData;
 
+    for (const k of Object.keys(accessTokenPayload)) {
+        if (accessTokenPayload[k] === null) {
+            delete accessTokenPayload[k];
+        }
+    }
```

Comment:
If `fetchAndSetClaim` returns not an "update" but the updated payload object then this could be removed. Although I still think it'd be good to match the behavior of merge.

Reply:
This can be removed from here since we should not have any null values anymore as build will not set them to null if fetchValue returns undefined